coalib/settings/ConfigurationGathering.py

```diff
@@ -147,7 +147,15 @@ def load_configuration(arg_list, log_printer, arg_parser=None):
     for item in list(cli_sections["default"].contents.pop("targets", "")):
         targets.append(item.lower())
 
-    if bool(cli_sections["default"].get("no_config", "False")):
+    # list of cli options which do not involve load config file
+    no_cofig_file_options = (
+        'no_config',
+        'show_bears',
+        'show_capabilities')
+
+    if any(
+            option in cli_sections["default"]
+            for option in no_cofig_file_options):
```

Comment:
Unfortunately, one line with this statement does not pass pep8 limit of 79 characters (83 instead).